test/TestData.hs

```diff
@@ -200,7 +200,7 @@ genDomain =
   where
     okay :: Text -> Bool
     okay t =
-      Text.all allowedChar t
+      Text.all atext t
```

Comment:
There is some flakiness in the tests. A couple of them result in `InvalidCharactersError` and `InvalidLengthError`. This change fixes the first one. Domain name is considered valid over `atext`. But property tests were allowing all ascii characters (Eg: `;`).